packages/dataviews/src/components/dataviews-item-actions/index.tsx

```diff
@@ -248,7 +248,7 @@ function CompactItemActions< Item >( {
 		<DropdownMenuV2
 			trigger={
 				<Button
-					size="compact"
+					size="small"
```

Comment:
@jameskoster Is the icon size change (from 32px to 24px) something to expect in this PR? It should affect all layouts.

Reply:
Good spot, no this should only affect the grid layout.

Reply:
Should i change it with styles to only impact on grid view?